api/client/modelconfig/modelconfig.go

```diff
@@ -103,3 +103,45 @@ func (c *Client) Sequences() (map[string]int, error) {
 	}
 	return result.Sequences, nil
 }
+
+// GetModelSecretBackend returns the secret backend name for the specified model,
+// returning an error satisfying [modelerrors.NotFound] if the model provided does not exist.
+func (c *Client) GetModelSecretBackend(ctx context.Context) (string, error) {
```

Comment:
The comments here say that [modelerrors.NotFound] is a possible error type that gets returned but we don't do any work here to produce that error.